grc/core/blocks/block.py

```diff
@@ -539,6 +541,26 @@ def is_virtual_sink(self):
     def is_virtual_source(self):
         return self.key == 'virtual_source'
 
+    def is_deprecated(self):
```

Comment:
Given the limited use of `self`, this could/should be a module-level function. I would also mark this protected (leading _) - no code outside of this module should use this.

Reply:
yep!

Reply:
problem: if we add that flag, we're suddenly back in a situation where it makes sense that this would be a method.

Reply:
This function does one thing - parse the categories. You can always write `or ... in self.flags` at the call-side

Reply:
If whenever I need the function I need to write `or some other criterion that's always the same`, then that should be part of the function, no? Principle of least surprise!